### ISSUE TITLE
cptIMSCobraExport - account for long file names

### ISSUE THREAD

**Issue by AronGahagan** [AronGahagan/cpt-dev#351]
## Problem:
Users who have not and/or cannot enable long file names have hit filename length limitation of 255 characters, causing an error on these lines:

```vba
cptIMSCobraExport_bas.bas:621:    Open ACTfilename For Output As #1
cptIMSCobraExport_bas.bas:1371:        Open ACTfilename For Output As #1
cptIMSCobraExport_bas.bas:2085:        Open ACTfilename For Output As #1
cptIMSCobraExport_bas.bas:3094:        Open ACTfilename For Output As #1
cptIMSCobraExport_bas.bas:3472:        Open ACTfilename For Output As #1
cptIMSCobraExport_bas.bas:3473:        Open RESfilename For Output As #2
cptIMSCobraExport_bas.bas:4035:        Open ACTfilename For Output As #1
cptIMSCobraExport_bas.bas:4377:        Open ACTfilename For Output As #1
cptIMSCobraExport_bas.bas:4378:        Open RESfilename For Output As #2
cptIMSCobraExport_bas.bas:4858:        Open ACTfilename For Output As #1
cptIMSCobraExport_bas.bas:5272:        Open ACTfilename For Output As #1
cptIMSCobraExport_bas.bas:5273:        Open RESfilename For Output As #2
```

## Solution:
TBD

## Todo:
- [x] assign this issue
- [x] label the issue for enhancement or bug
- [x] label the issue with codemodule (so fixes can be aggregated and hotfixed together)
- [x] checkout appropriate branch and create `topic` branch
- [x] design, code and test
- [x] update codemodule <cpt_version>x.y.z</cpt_version>
- [x] update CurrentVersions.xml **manually**
- [x] commit changes prefix comment with `Issue #XXX - `
- [x] merge `topic` into appropriate branch(es) and push
- [x] delete topic branch

**Comment by AronGahagan** [comment]
My suggestion would be: at the beginning of all routines, length of filename. If over a certain threshold that would result in a filename > 255 characters, prompt user once for an acronym and use that instead. Let me know if you want an assist with this.

**Comment by AronGahagan** [comment]
For the record: super long filenames are dumb.

**Comment by AronGahagan** [comment]
addressed in PR #413
